in_toto/runlib.py

```diff
@@ -101,11 +101,8 @@ def record_artifacts_as_dict(artifacts, exclude_patterns=None,
     Paths are normalized for matching and storing by left stripping "./"
 
     NOTE on exclude patterns:
-      - Uses Python fnmatch
-            *       matches everything
-            ?       matches any single character
-            [seq]   matches any character in seq
-            [!seq]  matches any character not in seq
+      - Uses PathSpec to compile gitignore-style patterns, making use of the 
+        GitWildMatchPattern class (registered as 'gitwildmatch')
```

Comment:
You changed the docs of an outdated `record_artifacts_as_dict` function, one that still used an `_apply_exclude_patterns` with `fnmatch`.

Please rebase you branch on top of [in-toto/in-toto@develop](https://github.com/in-toto/in-toto/tree/develop)! 

While even without rebasing a merge would actually pull your doc changes into the updated function, it is hard for me to review the docs if they don't correspond to the corresponding function.

Reply:
I understand. I have rebased my branch accordingly, so you should be able to review the docs now.